addons/flashdrive/functions/fnc_initInterface.sqf

```diff
@@ -11,72 +11,83 @@
 
 params ["_device", "_config"];
 
-private _flash_drives = {
-	params ["_target", "_player", "_params"];
-	_params params ["_config"];
-
-	private _actions = [];
-	private _class = configFile >> "CfgWeapons" >> "Item_FlashDisk_AE3";
-
-	private _add = {
-		params ["_target", "_player", "_params"];
-		([_target, _player] + _params) remoteExec ["AE3_flashdrive_fnc_connectFlashDrive", 2];
+if(!isDedicated) then
+{
+	// get or add parent action
+	private _parentActionPath = _device getVariable ["AE3_parentActionPath", []];
```

Comment:
Code duplication with addParentAction

Reply:
That's true. The USB interfaces are a special case. I think I need to pass additional variables to treat it like in the other cases. Also, for the future, we should rewrite our config-compile-init-system. It's quite a mess with duplicated logic in 3 different addons.